challenge-10/challenge-10.js

```diff
@@ -50,8 +69,19 @@ parâmetros;
 operador passado para a função "calculator", e passando para esse método
 os dois parâmetros da função de retorno de "calculator".
 */
-// ?
-
+	function calculator(operador) {
+		if( isOperatorValid(operador) === false ) {
+			return false;
+		} {
```

Comment:
haha entao mas é essa minha duvida
se em um estrutura if eu tenho um return eu nao preciso da palavra else certo? apenas uso a abertura de chaves? pq acho que já vi vc falando sobre isso em alguma aula ou to viajando?